Steer/DigitizerWorkflow/src/FITDigitizerSpec.cxx

```diff
@@ -207,6 +210,7 @@ o2::framework::DataProcessorSpec getFITT0DigitizerSpec(int channel)
              { "simFileS", VariantType::String, "", { "Sim (signal) input filename" } },
              { "pileup", VariantType::Int, 1, { "whether to run in continuous time mode" } } }
 
```

Comment:
See e.g. https://github.com/AliceO2Group/AliceO2/blob/d99d43997bfc2377bc52d738856894cf4af8d4f9/Steer/DigitizerWorkflow/src/TPCDigitizerSpec.cxx#L350 on using VariantType::Bool in similar situation. BTW, does FIT have triggered mode?